README.md

```diff
@@ -60,33 +62,14 @@ Further, prerequisites might be necessary for specific tasks. These are listed u
 
 ### Installation
 
-Download the installation and update script via:
-
-```
-curl -L -o install_or_update_exaslct.sh https://raw.githubusercontent.com/exasol/script-languages-container-tool/main/installer/install_or_update_exaslct.sh
-```
+Find the wheel package for a specific [release](https://github.com/exasol/script-languages-container-tool/releases) under assets.
 
-Before you continue with installation, please compute with the following command 
-the sha512 hash sum of the downloaded file and compare it with its 
-[checksum file](installer/checksums/install_or_update_exaslct.sh.sha512sum):
+Install the python package with `python3 -m pip install https://github.com/exasol/script-languages-container-tool/releases/download/0.9.0/exasol_script_languages_container_tool-0.9.0-py3-none-any.whl`
```

Comment:
This way we need to change it every release

Reply:
Changed the install path.